core/src/main/resources/hudson/model/Messages_bg.properties

```diff
@@ -166,38 +162,6 @@ CLI.wait-node-online.shortDescription=\
 CLI.wait-node-offline.shortDescription=\
  \u0418\u0437\u0447\u0430\u043a\u0432\u0430\u043d\u0435 \u043d\u0430 \u043c\u0430\u0448\u0438\u043d\u0430 \u0434\u0430 \u043d\u0435 \u0435 \u043d\u0430 \u043b\u0438\u043d\u0438\u044f.
 
-Computer.Caption=\
```

Comment:
this looks like a lot - is this no longer used and was just here because it was never deleted before?